src/components/chat/unlockProfile/UnlockProfileWrapper.tsx

```diff
@@ -13,7 +13,7 @@ interface IntroContainerProps {
   type?: UNLOCK_PROFILE_TYPE;
   showConnectModal?: boolean;
   description?: string;
-  onClose?: () => void;
+  handleClose?: () => void;
   closeIcon?: boolean;
```

Comment:
we don't need closeIcon, we can decide the close icon visibilty from the onClose prop.